content/applications/services/helpdesk/advanced/after_sales.rst

```diff
@@ -1,67 +1,236 @@
+
+.. image:: after_sales/after-sales-repair-reference.png
    :align: center
+   :alt: View of a repair reference page.
+
+If a product was specified in the :guilabel:`Product` field on the ticket, it will be automatically
+added to the :guilabel:`Product to Repair` field on the newly created :guilabel:`Repair Reference`.
+If applicable, add the :guilabel:`Sale Order` and :guilabel:`Return Order` from which the faulty
+product originates.
 
-Grant coupons from tickets
---------------------------
+Select the :guilabel:`Invoice Method`. If the customer will not be charged for this repair, this
+field can be left unchanged.
 
-First, be sure to have your *Coupon Program* planned in the *Sales* or *Website* application. Then,
-in *Helpdesk*, open your ticket, click on *Coupon*, and choose the respective one.
+If parts are required for the repair, they can be added in the :guilabel:`Parts` tab. Services can
+be added as product lines on the :guilabel:`Operations` tab. Additional information for the internal
+repair team can be added to the :guilabel:`Repair Notes` tab. Information for the customer can
+be added to the :guilabel:`Quotation Notes` tab, and will be automatically added to the PDF of the
+quotations generated from this :guilabel:`Repair Reference`.
 
-.. image:: after_sales/aftersales4.png
+.. image:: after_sales/after-sales-repair-tabs.png
    :align: center
+   :alt: View of the tabs on a repair reference page.
 
-Repairs from tickets
---------------------
+A :guilabel:`Repairs` smart button will be added to the :menuselection:`Helpdesk`
+:guilabel:`Ticket`, linking to the repair order.
 
-Clicking on *Repair* option, on your ticket page, a new repair order form is shown. Fill in the
-information as needed and choose the next step.
+.. image:: after_sales/after-sales-repair-smart-button.png
+   :align: center
+   :alt: View of smart buttons focusing on repair button.
+
+.. note::
+   If a user creates a repair order from a :menuselection:`Helpdesk` :guilabel:`Ticket`, they will
+   be able to access it through the ticket's :guilabel:`Repairs` smart button, or from the link in
+   the :guilabel:`Chatter`, even if they do not have access rights to the :menuselection:`Repair`
+   application.
+
+Create a field service task from a ticket
+=========================================
 
-.. image:: after_sales/aftersales5.png
+On-site interventions can be planned from a :guilabel:`Ticket` and managed through the
+:menuselection:`Field Service` application. Customers with portal access will be able to track the
+progress of a :guilabel:`Field Service` task just as they would a :menuselection:`Helpdesk`
+:guilabel:`Ticket`.
+
+To create a new task, go to a :menuselection:`Helpdesk` :guilabel:`Ticket` and click
+:guilabel:`Create Task`. Confirm the :guilabel:`Field Service Project` and :guilabel:`Worksheet
+Template`, then click :guilabel:`Create Task`. A new task will be created within the designated
+:guilabel:`Field Service` project.
```

Comment:
tag formatting
what are **Field Service Project** and **Worksheet**, exactly?
(hint: when I look at the _pop-up window_ labeled **Create a Field Service task**,  this is not what those _form labels_ say)

Reply:
Updated 👍 I changed out the screenshot and removed the mention of `Worksheet templates` because after confirming in runbot and my demo database, that is actually not a default configuration. I'd rather not go too deep into explaining Field Service configurations (especially since the worksheet templates are not particularly relevant to this process )